### PR TITLE
Fixing map tests

### PR DESCRIPTION
**Pull request by oldergod**

### CODE REVIEW

File: wire-library/wire-tests/src/commonTest/shared/json/all_64_max_proto3.json [hunk 1]
```diff
@@ -44,5 +44,20 @@
     "9223372036854775807",
     "9223372036854775807"
   ],
-  "oneofInt64": "9223372036854775807"
+  "oneofInt64": "9223372036854775807",
+  "mapInt64Int64": {
+    "9223372036854775807": "9223372036854775807"
+  },
+  "mapInt64Uint64": {
+    "9223372036854775807": "9223372036854775807"
```
Comment (swankjesse) [inline wire-library/wire-tests/src/commonTest/shared/json/all_64_max_proto3.json:52]: this is the max signed long . . . use the max unsigned long to exercise the difference?

File: wire-library/wire-tests/src/commonTest/shared/json/all_64_min_proto3.json [hunk 2]
```diff
@@ -44,5 +44,20 @@
...
+    "-9223372036854775808": "-9223372036854775808"
+  },
+  "mapInt64Uint64": {
+    "-9223372036854775808": "9223372036854775808"
+  },
+  "mapInt64Sint64": {
+    "-9223372036854775808": "-9223372036854775808"
+  },
+  "mapInt64Fixed64": {
+    "-9223372036854775808": "9223372036854775808"
```
Comment (swankjesse) [inline wire-library/wire-tests/src/commonTest/shared/json/all_64_min_proto3.json:58]: I would have expected this to be 0? unless we’re testing that it is backwards-compatible

Comment (oldergod) [reply wire-library/wire-tests/src/commonTest/shared/json/all_64_min_proto3.json:58]: This is the JSON generated by protoc

Comment (oldergod) [reply wire-library/wire-tests/src/commonTest/shared/json/all_64_min_proto3.json:58]: Right, will update what we test in followup

**Pull request merged**